layers/+source-control/git/README.org

```diff
@@ -136,6 +159,46 @@ To enable it you have to add the following lines to your
 #+END_SRC
 
 ** Forge
+Magit Forge can view and create issues & pull requests with forges
+(e.g. GitHub, GitLab)
+
+Magit Forge requires a username for the respective forge and will prompt for a
+username if not found, writing it to  =~/.gitconfig=
+
+Explicitly define a forge identity using the `git` command.
+
+For GitHub:
+
+#+begin_src shell
+  git config --global github.name "username"
```

Comment:
Should this be `github.user` or `github.name`? I find on [this page](https://practical.li/spacemacs/source-control/forge-configuration.html#set-the-username-for-the-service-used) it's `github.user`

Reply:
I also believe it's 'github.user'